Libraries/Utilities/Platform.ios.js

```diff
@@ -14,6 +14,7 @@
 
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => obj.ios,
```

Comment:
We could probably annotate it like `select<T>(obj: {ios: T}): T` if that's something useful